### ISSUE TITLE
Possible conflict with Incontrol mod

### ISSUE THREAD

**Issue by Morozzs** [Sunconure11/dungeon-mobs#32]
Hi. I use Incontrol to configure mobs spawn rate, and while other mods work as intended, I can't do anything with dungeon mobs. They spawn in very small numbers regardless of spawnrate in either Dungeon Mobs or Incontrol config.

**Comment by BlackJar72** [comment]
Did you set spawn anywhere to true?  If not try that.  If so I'm not sure what the problem is -- they all have code added to the mobs canSpawnHere() method, but that should turn it off.  For example:

    B:BeholderSpawnAnywhere=true

Should make beholders spawn like vanilla mobs.

**Comment by Morozzs** [comment]
Yes, I did it for a test with a Rust Monster. The only change I noticed is that they started spawning on the surface, nothing else.

**Comment by BlackJar72** [comment]
I just set beholder to 200 spawn chance spawn anywhere, and got tons.  I don't know anything about how the other mod actually works, but you may need to set spawn rules in both to be safe and use spawn anywhere on any mob you want Incontrol to control.

**Comment by Sunconure11** [comment]
At some point, I will return to work on this a bit more, but I've got to get my main mod into closed beta first.

**Comment by BlackJar72** [comment]
I may look at some of this in a couple weeks, though I don't really plan on acting as a major developer in the future of this in the future.

**Comment by Morozzs** [comment]
> I just set beholder to 200 spawn chance spawn anywhere, and got tons. I don't know anything about how the other mod actually works, but you may need to set spawn rules in both to be safe and use spawn anywhere on any mob you want Incontrol to control.

I tried fiddling with both mods settings, but when Incontrol is installed neither will do a thing. In terms of spawnrate, specifically: "spawn anywhere" actually works. I've even tried it with only two mods installed just to be safe.
Anyway, I posted it on Incontrol github as well, perhaps there is some problem with it's configuration. Hope this problem will be resolved, cause I don't know a reliable spawncontrol mod for 1.12.2 other than this one.
